src/python/pants/backend/project_info/tasks/export.py

```diff
@@ -104,11 +104,8 @@ def register_options(cls, register):
              help='Causes libraries with javadocs to be output.')
     register('--sources', type=bool,
              help='Causes sources to be output.')
-    # Required by IvyTaskMixin.
-    # TODO: Remove this once IvyTaskMixin registers an --ivy-jvm-options option.
```

Comment:
Coursier should probably have such an option on its subsystem (and not on the Tasks that consume it).